src/h5web/dataset-visualizer/VisDisplay.tsx

```diff
@@ -42,7 +28,7 @@ function VisDisplay(props: Props): JSX.Element {
   return (
     <VisProvider mapping={dimensionMapping} dataset={dataset}>
       <DimensionMapper activeVis={activeVis} onChange={setMapping} />
-      <ActiveVis />
+      <Component />
```

Comment:
Minor complaint here: I would rename it `VisComponent` but it can be kept as `Component` in the `VIS_DEFS`. This way, we can understand what the component `VisDisplay` returns without searching what `Component` refers to in the code.

In other words, the destructuring above would be `const { Component:  VisComponent } = VIS_DEFS[activeVis];`